SnaffCore/SnaffCon.cs

```diff
@@ -88,15 +88,18 @@ public void Execute()
             statusUpdateTimer.Elapsed += TimedStatusUpdate;
             statusUpdateTimer.Start();
 
-            // if we haven't been told what dir or computer to target, we're going to need to do share discovery. that means finding computers from the domain.
-            if (MyOptions.PathTargets == null && MyOptions.ComputerTargets == null)
+
+            // If we want to hunt for user IDs, we need data from the running user's domain.
+            // Future - walk trusts
+            if ( MyOptions.DomainUserRules)
             {
-                DomainDiscovery();
+                DomainUserDiscovery();
             }
-            // if we've been told what computers to hit...
-            else if (MyOptions.ComputerTargets != null)
+
+            // If we weren't given path targets then assume we want to do network discovery (even if it's just to get the DFS dedupe benefits
```

Comment:
I'm concerned that users will assume that providing a list of computer targets means that snaffler will skip comms with AD. I'd rather force them to explicitly enable AD discovery if they've provided a list of path or computer targets.